package.json

```diff
@@ -14,9 +14,10 @@
     "test:current": "tfm-test --watch",
     "publish-coverage": "tfm-publish-coverage",
     "stories": "tfm-stories",
+    "fix-stories": "./script/npm-fix-foreman-stories.sh",
```

Comment:
how about `install:stories` ?
```suggestion
    "install:stories": "./script/npm-fix-foreman-stories.sh",
```

Reply:
does that have some special meaning? or just convention?

Reply:
Just a convention

Reply:
We could also move that to the actual `stories` entry, and check for the existence of `node_modules/@theforeman/stories/node_modules` in `npm-fix-foreman-stories.sh`, so that it only gets called once on the first `stories` invocation?

Reply:
+1 for that

Reply:
Updated!